de.metas.contracts/src/main/java/de/metas/contracts/IFlatrateBL.java

```diff
@@ -89,17 +93,30 @@
 	void updateEntry(I_C_Flatrate_DataEntry dataEntry);
 
 	/**
-	 * Create a new flatrate term using the given term as template. The new term's C_Year will be the year after the given term's C_Year.
-	 *
-	 * <b>IMPORTANT:</b> the method might set the given term's C_FlatrateTerm_Next_ID, but won't save it!
+	 * term to extend
+	 * forceExtend - will create a new term, even if the given <code>term</code> has <code>IsAutoRenew='N'</code>
+	 * forceComplete - will complete a new term (if one has been created), even if it has <code>IsAutoComplete='N'</code>
```

Comment:
here is an outdated (a.k.a. lying) comment

Reply:
yes, right!
Thx!